Azure/azure-cli
# {Synapse} Fix test fail in yum package

**Description**

`test_notebook` fails in [Test Yum Package](https://dev.azure.com/azure-sdk/public/_build/results?buildId=837520&view=logs&j=1e6fba43-bdcf-51f0-a062-3712c33fbb51&t=a1e787eb-ec67-5c3d-79ec-0c68f4f59d65)

It might caused because the operation system ADO used can't deal with `ipynb` file. This PR changed `ipynb` file in test to `json` file.

**Testing Guide**

`azdev test test_notebook`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
